source/material_model/multicomponent.cc

```diff
@@ -170,14 +176,16 @@ namespace aspect
   {
     ASPECT_REGISTER_MATERIAL_MODEL(Multicomponent,
                                    "multicomponent",
-                                   "This model is for use with an arbitrary number of compositional fields, where each field"
-                                   " represents a rock type which can have completely different properties from the others."
-                                   " However, each rock type itself has constant material properties.  The value of the "
-                                   " compositional field is interpreted as a volume fraction. If the sum of the fields is"
-                                   " greater than one, they are renormalized.  If it is less than one, material properties "
-                                   " for ``background mantle'' make up the rest. When more than one field is present, the"
-                                   " material properties are averaged arithmetically.  An exception is the viscosity,"
-                                   " where the averaging should make more of a difference.  For this, the user selects"
-                                   " between arithmetic, harmonic, geometric, or maximum composition averaging.")
+                                   "This model is for use with an arbitrary number of compositional fields, where each field "
+                                   "represents a rock type which can have completely different properties from the others. "
+                                   "However, each rock type itself has constant material properties.  The value of the "
+                                   "compositional field is interpreted as a mass fraction. If the sum of the fields is "
+                                   "greater than one, they are renormalized.  If it is less than one, material properties "
+                                   "for ``background mantle'' make up the rest. When more than one field is present, the "
+                                   "material properties are averaged arithmetically by mass fraction (for specific heat), "
+                                   "or volume fraction (for density, thermal expansivity and compressibility). "
```

Comment:
This model doesn't have compressibility.

Reply:
Thanks! I think it's still worth specifying how the compressibilities would be averaged if they were not zero, but I've added a few extra words to make it clear that this model is incompressible.